stack

```diff
@@ -5,6 +5,25 @@ IFS=$'\n\t'
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 OS_ARCH=$(uname -m | awk '{if ($0 ~ /arm64|aarch64/) print "arm64"; else if ($0 ~ /x86_64|amd64/) print "amd64"; else print "unsupported_arch"}')
 
+############################################################################
+# docker compose 
+############################################################################
+function compose-init() {
+  chain-clean
+  rm -Rf ./data/chain
+  ADMIN_ADDRESS=${@:-"0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"} \
+  docker compose -f ./docker/docker-compose.dev.yml up chain -d
+  chain-boot
+}
+
+function compose-up() {
+  ADMIN_ADDRESS=${@:-"0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"} \
```

Comment:
Is the admin address required here? It is passed to the `chain` when it executes but the rest of the processes don't need it.